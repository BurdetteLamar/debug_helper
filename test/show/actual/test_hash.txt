---
Name: hash
Class: Hash
Size: 2
Default: nil
Pairs:
  0:
    Key:
      Class: Symbol
      Size: 1
      Value: :c
    Value: 2 (Fixnum)
  1:
    Key:
      Class: Symbol
      Size: 1
      Value: :d
    Value: 3 (Fixnum)
