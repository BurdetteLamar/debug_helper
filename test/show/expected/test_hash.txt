Label: hash
Count: 2
Pair 0:
  Key (Symbol) :a
  Value (Fixnum) 0
Pair 1:
  Key (Symbol) :b
  Value (Fixnum) 1

